app/models/exchange_rate_country_currency.rb

```diff
@@ -13,4 +13,29 @@ def validate
     validates_presence :currency_code
     validates_presence :validity_start_date
   end
+
+  def self.live_currency_codes
```

Comment:
Is this actually used?

Reply:
Technically not since last iteration but kept it in as could be useful but appreciate could be taken out